de/_posts/2019-09-21-code-and-cake.markdown

```diff
@@ -0,0 +1,21 @@
+---
+tags: intermediate code-cake
+categories: de events
+i18n_key: code_cake_2019_09
+---
+
+## Lerne noch mehr Programmieren ♥️
+
+Egal ob du an einem Beginner Workshop teilgenommen hast oder dir selber programmieren beibringst, Code & Cake Workshop ist der perfekte Tag für dich um weiter zu lernen.
+
+Nutze diese Stunden, um dich tiefer mit einem Tutorial zu beschäftigen, einem Coach deine Fragen zu stellen oder mit deinem Projekt fortzufahren, an dem du bereits arbeitest.
+
+Du kannst zusammen mit deiner Projektgruppe vorbeikommen (auch mit euren Coaches, wenn ihr wollt), mit FreundInnen oder alleine. Du kannst mit anderen zusammen arbeiten, aber du musst nicht.
```

Comment:
Gendering texts is super confusing and hard sometimes. But also this needs some kind of consistency. I would recommend using either the underline version (Freund_innen) or the asterix version (Freund* innen) in order to include all genders in between the binary ones. Follow the chosen version throughout the whole website for consistence. For example in line 17 also use jede_r or jede*r etc.
Btw. because of github markdown I had to make a space after the first asterix in order to not just markdown the following characters as italic.

Reply:
Typing a backslack like so `Friend\*innen` can be used, which outputs to: Friend\*innen. 

Stylistically I have also seen `Friend:innen`, which is the difference between all three?

Reply:
I think in German at least I have only seen * and _ in academic texts not the : version. There is not a formal difference between the two as far as I know.

Reply:
The TAZ uses both `*innen` and `:innen` frequently so was curious https://taz.de/!s=%253Ainnen/